alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureMonitorRestApiIT.kt

```diff
@@ -195,19 +215,22 @@ class SecureMonitorRestApiIT : AlertingRestTestCase() {
                 )
             )
             userClient?.makeRequest("POST", ALERTING_BASE_URI, emptyMap(), monitor.toHttpEntity())
-            fail("Expected 403 Method FORBIDDEN response")
```

Comment:
Why is this removed? We need this to throw an assertion failure if it doesn't throw an exception

Reply:
added